apps/business_central/exports/journal_entry/models.py

```
@@ -94,42 +103,44 @@
-                    'account_id': account_id,
-                    'account_type': account_type,
-                    'document_number':  document_number,
-                    'accounts_payable_account_id': account.destination_account.destination_id,
-                    'comment': comment,
-                    'workspace_id': accounting_export.workspace_id,
-                    'invoice_date': invoice_date,
-                    'description': lineitem.purpose if lineitem.purpose else None
-                }
-            )
-            journal_entry_lineitems.append(journal_entry_lineitems_object)
+        category = lineitem.category if (lineitem.category == lineitem.sub_category or lineitem.sub_category == None) else '{0} / {1}'.format(lineitem.category, lineitem.sub_category)
```

Ruff (0.7.0)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)